src/lib/markbind/src/parser.js

```diff
@@ -139,32 +123,12 @@ class Parser {
     switch (node.name) {
     case 'md':
       node.name = 'span';
-      cheerio.prototype.options.xmlMode = false;
       node.children = cheerio.parseHTML(md.renderInline(cheerio.html(node.children)), true);
-      cheerio.prototype.options.xmlMode = true;
       break;
     case 'markdown':
       node.name = 'div';
-      cheerio.prototype.options.xmlMode = false;
       node.children = cheerio.parseHTML(md.render(cheerio.html(node.children)), true);
-      cheerio.prototype.options.xmlMode = true;
-      break;
-    case 'panel': {
```

Comment:
this is moved to `componentPreprocessor` L149 - L155.

This chunk does the rewriting of `src="topic1.html"` into `src="topic1._include_.html"`, so it's still necessary

The main difference is the removal of the `fileExists` check, which already wasn't necessary before (there's a similar check done in `componentPreprocessor`)
The second difference would be `path.dirname(path.join('{{hostBaseUrl}}', path.relative(config.rootPath, src)))` using `config.baseUrl` directly instead of `{{ hostBaseUrl }}`, since we no longer have / render `{{ hostBaseUrl }}`